.gitignore

```diff
@@ -7,8 +7,13 @@
 Files.App.OpenDialog*.dll
 Files.App.SaveDialog*.dll
 Files.App.Launcher.exe
+CustomOpenDialog*.dll
+CustomSaveDialog*.dll
+FilesLauncher.exe
```

Comment:
Why is this excluded?

Reply:
Yes, those aren't used any more.

Reply:
Yeah I was going to say these used to be excluded